common/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
+    <style name="AppTheme" parent="NavigationViewDark">
```

Comment:
This is the fix for crash with navigation right?

Reply:
Have you submitted an issue to navigation repo for that?

Reply:
nav docs have such info. 
I submitted an issue to update `RouteListener` docs.

Reply:
Great!